Gem/Code/Source/Automation/ScriptReporter.cpp

```diff
@@ -1081,4 +1094,33 @@ namespace AtomSampleViewer
         }
 
     }
+
+    void ScriptReporter::ExportTestResults()
+    {
+        uint32_t totalAssertCount = 0;
+        uint32_t totalErrorCount = 0;
+        uint32_t totalWarningCount = 0;
+        uint32_t totalScreenshotErrorCount = 0;
+        uint32_t totalScreenshotWarningCount = 0;
+        
+        for (ScriptReport& scriptReport : m_scriptReports)
+        {
+            totalAssertCount += scriptReport.m_assertCount;
+            totalErrorCount += scriptReport.m_generalErrorCount;
+            totalWarningCount += scriptReport.m_generalWarningCount;
+            totalScreenshotErrorCount += scriptReport.m_screenshotErrorCount;
+            totalScreenshotWarningCount += scriptReport.m_screenshotWarningCount;
+        }
+
+        std::ofstream exportResultsFile;
+        exportResultsFile.open("exported_results.txt");  // This will save to the default bin you built with.
+        exportResultsFile << AZStd::string::format("Asserts: %s \n", std::to_string(totalAssertCount).c_str()).c_str();
```

Comment:
Prefer not to convert every number to a string and just serialize it as an int instead (using the `%i` format specifier)